src/components/containers/YourClips/YourClips.js

```diff
@@ -1,10 +1,25 @@
 import React, { Component } from 'react';
 import './YourClips.css';
+import downloadcloud from '/Awesome API Github/stt-tooling/src/cloud-upload-2.png'
+import mp3file from '/Awesome API Github/stt-tooling/src/mp3_file.png'
```

Comment:
I am confused by where these imports are coming from. We should just create an `images` directory at `stt-tooling/public` and put our images there.